src/atsu/common/src/databus/Fms.ts

```diff
@@ -0,0 +1,89 @@
+import { AtsuStatusCodes } from '@atsu/common/AtsuStatusCodes';
+import { FansMode } from '@atsu/common/com/FutureAirNavigationSystem';
+import {
+    AtisMessage,
+    AtisType,
+    AtsuMessage,
+    CpdlcMessage,
+    DclMessage,
+    FreetextMessage,
+    OclMessage,
+    WeatherMessage,
+} from '@atsu/common/messages';
+import { PositionReportData, Waypoint } from '@atsu/common/types';
+
+export interface AtsuFmsRegisterMessages<T> {
+    requestId: number;
+    messages: T[];
+}
+
+export interface FmsRouteData {
+    lastWaypoint: Waypoint;
+    activeWaypoint: Waypoint;
+    nextWaypoint: Waypoint;
+    destination: Waypoint;
+}
+
+export interface AtsuFmsMessages {
```

Comment:
I think we should make the messages sent from ATSU -> FMS be a separate interface than the messages sent from FMS -> ATSU.

Reply:
fixed